src/Microsoft.AspNet.Mvc.Core/Rendering/ViewEngine/IView.cs

```diff
@@ -5,8 +5,21 @@
 
 namespace Microsoft.AspNet.Mvc.Rendering
 {
+    /// <summary>
+    /// Specifies the contract for a view.
+    /// </summary>
     public interface IView
     {
+        /// <summary>
+        /// Gets the path of the view as resolved by the <see cref="ViewResult"/>.
```

Comment:
doesn't this come from the view engine? there's nothing stopping you from using this outside of `ViewResult`